docs/user_guide/index.rst

```diff
@@ -226,30 +226,32 @@ on a predetermined set while remaining aware of the others.
 
 This audit should be used by experts with a deep understanding of the
 model and the context in which the predictions are used. Even when a
-metric is flagged as failing in the fairness audit, there might be
-context that explains and even predicts the difference. Like many
+metric is flagged as a deviation in the fairness audit, the context 
+might that explains or even predict the difference. Like many
 concepts, a single parity concept can have several different names;
 notably, parity of true positive rate is equal opportunity, parity of
 false positive rate is predictive equality, and parity of predictive
 prevalence is demographic parity.
 
-An Aequitas audit gives an overview of parity across all defined groups
-for each cohort attribute. By default, the majority group is the
+A fairness audit gives an overview of parity across all defined groups
+for each cohort attribute. The majority group is the
 baseline and a statistic for all observations in the other groups is
-compared. A fairness threshold such as 125% is then used to classify the
-ratio of each group to the reference. If any group performs differently,
-above (125% in our example) or below (80%) then it is considered a
-failure for that cohort/metric.
-
-The visualization is a table showing the overall pass/fail, an ordered
-list of circles representing the groups, and a bar representing the
-percentage of the population data in reference. Note that comparison
+compared. A fairness threshold such as 25% is then used to classify the
+ratio of each group to the reference. The metric of interest is calculated on the default
+group and the cohort under comparison. The resulting ratio (comparison/default) is then 
+compared against the allowed bounds determined by the fairness threshold.
+The bound determined by 1 + threshold above, and 1 / (1 + threshold) below,
+so that a fairness threshold of 0.25 sets the upper bound at 1.25 times larger,
+or a 25% increase in the metric. Since the lower bound is checked
+with the recriprical, this would result in a 20% decrease.
```

Comment:
"reciprocal". You may also want to mention that there could be floor and ceiling effects if the referent values are near extreme values. You can't be more than 20% above 83.3%., and although you can be 50% under 1%,, it's most likely not concerning.

Reply:
fixed the spelling, I didn't want to mention the floor/ceiling effects since I couldn't come up with good wording around this. There is a number of cases where a 20% difference is enormous (20% difference in Temperature) and there are cases where is is less worrisome (20% on Systolic BP)